library/docs/changelog.md

```diff
@@ -8,6 +8,7 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ### Added
 - Gradle plugin generates `sourcedBuilder()` extension function for `SourcedFeatureStorage` that returns custom builder with steps for each source. This makes changes to sources compile time safe.
+- Show feature's supervisor option in inspector (https://github.com/MiSikora/laboratory/issues/95).
```

Comment:
I assumed it would get automatically shortened when viewed on GitHub. I wonder if it'll still work with this:
[https://github.com/MiSikora/laboratory/issues/95](https://github.com/MiSikora/laboratory/issues/95)

Reply:
It is a GH feature, you can basically write `#95` and it will work as well. But the changelog is also deployed to the website so it needs formatting.

Reply:
f705f29a32d86de4f8aabdfdb9bf9ab4a20861f8